packages/sdk/src/modules/Pools.ts

```diff
@@ -229,5 +232,31 @@ export function withPools<TBase extends IonicBaseConstructor>(Base: TBase) {
 
       return [...filteredPools, ...whitelistedPools].filter((p) => !!p) as IonicPoolData[];
     }
+
+    async isAuth(pool: string, market: string, role: Roles, user: string) {
+      const authRegistry = this.createAuthoritiesRegistry();
+      const poolAuthAddress = await authRegistry.callStatic.poolsAuthorities(pool);
+
+      if (poolAuthAddress === constants.AddressZero && this.chainId !== SupportedChains.neon) {
```

Comment:
This is wrong. If the chainId  == neon.chainId, then `isAuth` must return True.

Reply:
fixed